modules/web/js/ballerina/components/struct-definition.jsx

```diff
@@ -76,9 +76,18 @@ class StructDefinition extends React.Component {
         this.validateIdentifierName(identifier);
         this.props.model.addVariableDefinitionStatement(bType, identifier, defaultValue);
     }
+    /**
+     * Handle the default value if the data type is a string
+     * @param {string} dataType - Data type of the new struct
+     * @param {string} value - Value of the new identifier
+     */
     addQuotesToString(dataType, value) {
         if (dataType === 'string' && !/".*?"$/.test(value)) {
-            return `"${value}"`;
+            if (value !== '') {
+                return `"${value}"`;
+            } else if (value === '') {
```

Comment:
this is the negate of the if condition. we can just have if-else. Better to have the empty check first though for readability.